themes/default/assets/config/postcss.config.js

```diff
@@ -4,95 +4,92 @@
  * at the end to do more transformations.
  */
 
-// Note whether we're in "production" mode, so we can avoid doing post-processing and
-// optimization during development.
-const isProduction = process.env.NODE_ENV === "production";
```

Comment:
Minification/purge only takes a few seconds in pulumi-hugo now, so this enables it in development as well.

Reply:
🎉 woohoo!